src/Components/Facility/LocationManagement.tsx

```diff
@@ -79,16 +79,17 @@ const Location = ({
 }: LocationModel) => (
   <div className="flex h-full w-full flex-col rounded border border-gray-300 bg-white p-6 shadow-sm transition-all duration-200 ease-in-out hover:border-primary-400">
     <div className="flex-1">
-      <div className="flex w-full items-center justify-between gap-2">
-        <div className="flex items-center gap-3">
+      <div className="flex w-full items-start justify-between gap-2">
+        <div className="flex items-start gap-3">
           <p
-            className="break-words text-xl font-medium"
+            style={{ wordBreak: "break-word" }}
```

Comment:
Use tailwindcss to achieve the same

Reply:
used the `break-all` utility from tailwind instead.